cmd/image-processing/main_test.go

```diff
@@ -22,6 +24,7 @@ import (
 	containerreg "github.com/google/go-containerregistry/pkg/v1"
 	"github.com/google/go-containerregistry/pkg/v1/empty"
 	"github.com/google/go-containerregistry/pkg/v1/remote"
+	shipwrightv1alpha1 "github.com/shipwright-io/build/pkg/apis/build/v1alpha1"
```

Comment:
Nit: In light of https://github.com/shipwright-io/build/issues/1485, a version less import name maybe?

Reply:
Changed it to `buildapi`